Dockerfiles/Dockerfile.cvp

```diff
@@ -0,0 +1,16 @@
+FROM registry.fedoraproject.org/fedora:33
+WORKDIR /project
+ARG OPERATOR_SDK_VERSION=v1.4.0
+ARG OPERATOR_TEST_PLAYBOOKS_TAG=v1.0.11
+RUN export ARCH=$(case $(arch) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(arch) ;; esac);\
+    export OS=$(uname | awk '{print tolower($0)}');\
+    export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/$OPERATOR_SDK_VERSION/;\
+    curl -L -o /usr/local/bin/operator-sdk ${OPERATOR_SDK_DL_URL}/operator-sdk_${OS}_${ARCH} \ 
+    && chmod a+x /usr/local/bin/operator-sdk && \
+    mkdir /project/output && \
+    dnf install --setopt=install_weak_deps=False -y git-core ansible python3-pip mlocate && \
+    dnf clean all && \
+    pip3 install operator-courier && \
+    git clone --branch $OPERATOR_TEST_PLAYBOOKS_TAG https://github.com/redhat-operator-ecosystem/operator-test-playbooks.git
+COPY ./entrypoint_cvp.sh /project/entrypoint_cvp.sh
+ENTRYPOINT ["/project/entrypoint_cvp.sh"]
```

Comment:
Why do we need entrypoint? We can manually specify file when running container

Reply:
IMO Entrypoint acts as starting point execution. It can be extended in near future to add multiple tests. 
we can just add a function in a shell script to add multiple tests.